### PR TITLE
Issue #3014655: Check if node is not empty before attempting to translate it

### PR DESCRIPTION
**Pull request by aaronferris**

### CODE REVIEW

**Comment by aaronferris** [comment]
Issue was local.

File: src/Service/Core.php [hunk 1]
```diff
@@ -222,21 +222,23 @@ public function getAppNode($module, $language = NULL) {
     $nid = end($nids);
     $node = $node_storage->load($nid);
 
-    try {
+    if (!empty($node)) {
```
Comment (malcomio) [inline src/Service/Core.php:225]: we should move this check earlier - no need to try the load if `$nid` is empty

**Comment by malcomio** [comment]
https://www.drupal.org/project/spalp/issues/3014655

**Comment by malcomio** [comment]
replaced by #29

**Pull request closed without merging**